src/data/faq.json

```diff
@@ -158,11 +158,23 @@
                         "anchor": "cert_eu_travel",
                         "active": true,
                         "textblock": [
+                            "<b>Update</b><br/>Please note that the behavior of the certificate validity check has changed with version 2.8. For more information on this change, please see the FAQ entry: <a href='#dcc_validity_check_change' target='_blank' rel='noopener noreferrer'>Why does the Corona-Warn-App on version 2.8 show that my certificate couldn't be validated although the previous version showed my certificate as valid?</a><hr>",
                             "Starting with version 2.6, you can check before travelling whether your certificates (test, recovery and/or vaccination certificate) are valid in a selected country at the time of your trip. The Corona-Warn-App takes into account the applicable entry rules of the selected travel country and compares them with various parameters of the certificate, such as the date and type of test, test center or date of vaccination.",
                             "Every European country that supports EU digital COVID certificates has the ability to upload rules that the Corona-Warn-App can match for verification. At the start of this new feature, Ireland, Lithuania, Luxembourg, the Netherlands, Slovenia and Spain, together with Germany, have uploaded rules, with more countries to follow in the coming weeks.",
                             "If the app shows after checking that there are no entry rules available, you can find out about the rules at <a href='https://reopen.europa.eu/en' target='_blank' rel='noopener noreferrer'>https://reopen.europa.eu/en</a>."
                         ]
                     },
+                    {
+                        "title": "Why does the Corona-Warn-App on version 2.8 show that my certificate couldn't be validated although the previous version showed my certificate as valid?",
+                        "anchor": "dcc_validity_check_change",
+                        "active": true,
+                        "textblock": [
+                            "With version 2.8 the behaviour of the certificate validity check has changed. Entry rules for travellers have been set up by all EU member countries, but until today only a few countries provided entry rules in machine-readable format which Corona-Warn-App can make use of. Before version 2.8, when there were no rules available to be downloaded for a certain country, after a check Corona-Warn-App displayed 'Certificate Valid' and explained in the following text, that for this country no rules were provided/could have been checked. 'Certificate Valid' referred to 'Technical Validity' in this case (more information on what the 'Technical Validity' is can be found in <a href='#eu_dcc_validity' target='_blank' rel='noopener noreferrer'>this</a> FAQ entry).",

```

Comment:
Use USA English spellings to conform to app standards.

behaviour => behavior
https://dictionary.cambridge.org/dictionary/english/behaviour

travellers => travelers
https://dictionary.cambridge.org/dictionary/english/traveller